manifests/latest/vsphere-7.0u1/vanilla/deploy/vsphere-csi-controller-deployment.yaml

```diff
@@ -143,7 +145,7 @@ metadata:
   name: csi-feature-states
   namespace: kube-system
 ---
-apiVersion: storage.k8s.io/v1beta1
+apiVersion: storage.k8s.io/v1 # For k8s 1.17 or lower use storage.k8s.io/v1beta1
```

Comment:
Similar to this, what about adding a comment on above sidecars specifying what to use for k8s version that are not supported by that sidecar

Reply:
The sidecar containers documentation itself have the supportability for k8s versions.